profiles/schema.py

```
@@ -1554,12 +1550,10 @@ def mutate_and_get_payload(cls, root, info, **input):
         try:
             profile = Profile.objects.get(user=info.context.user)
         except Profile.DoesNotExist:
-            raise ProfileDoesNotExistError("Profile does not exist")
+            raise ProfileDoesNotExistError(PROFILE_DOES_NOT_EXIST_MESSAGE)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)